proto/osmosis/mint/v1beta1/genesis.proto

```diff
@@ -1,16 +1,22 @@
 syntax = "proto3";
-package cosmos.mint.v1beta1;
+package osmosis.mint.v1beta1;
 
 import "gogoproto/gogo.proto";
-import "cosmos/mint/v1beta1/mint.proto";
+import "osmosis/mint/v1beta1/mint.proto";
 
-option go_package = "github.com/cosmos/cosmos-sdk/x/mint/types";
+option go_package = "github.com/c-osmosis/osmosis/x/mint/types";
```

Comment:
why isn't this osmosis-labs?

Reply:
Because we have to change the package in go.mod here: https://github.com/osmosis-labs/osmosis/blob/main/go.mod#L1

But I don't want to do that until the end as to not break everyone's in-flight PRs.